src/server.rs

```diff
@@ -4564,8 +4553,11 @@ fn find_node(
 struct FunctionResult {
     name: String,
     package: String,
+    #[allow(dead_code)]
```

Comment:
Did something change that required these to be added? How was this not being caught before?

Reply:
I am using nightly rust locally which, being newer, might catch more things :shrug: